### PR TITLE
feat(forms/TUI-336): add updating feedback

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

When a trigger update fields (using `options`) we need to give a feedback to the enduser

**What is the chosen solution to this problem?**

add heartbeat on corresponding fields
add circularprogress to the clicked button

demo http://2021.talend.surge.sh/forms/?selectedKind=Core%20concepts&selectedStory=Core%20Concept%20updating

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module.exports = function(api) {
+module.exports = function babel(api) {
```
Comment (jmfrancois) [inline babel.config.js]: this to fix lint on the whole function

File: packages/containers/sandbox/mockBackend/kit.js [hunk 2]
```diff
@@ -87,6 +88,7 @@ function suggestionBig() {
 		function onEnd() {
 			console.log('onEnd', body);
 			cache.photos = {
+				cacheable: true,
```
Comment (jmfrancois) [inline packages/containers/sandbox/mockBackend/kit.js]: fix the trigger return so we do not have loading again and again on the same field

File: packages/theme/src/theme/_forms.scss [hunk 3]
```diff
@@ -57,7 +57,7 @@ form {
 
 		label {
 			&.control-label {
-				color: $silver-chalice;
+				color: $dove-gray;
```
Comment (jmfrancois) [inline packages/theme/src/theme/_forms.scss]: to respect the guideline

File: packages/forms/src/UIForm/Widget/Widget.component.js [hunk 4]
```diff
@@ -20,6 +20,15 @@ function getWidget(displayMode, widgetId, customWidgets) {
 	return widget;
 }
 
+function isUpdating(updatingKeys = [], key) {
+	if (!updatingKeys.length || !key) {
```
Comment (sgendre) [inline packages/forms/src/UIForm/Widget/Widget.component.js]:
```suggestion
	if (updatingKeys.length === 0 || !key) {
```

File: packages/forms/src/UIForm/fields/Select/Select.component.js [hunk 5]
```diff
@@ -30,13 +39,14 @@ export default function Select({ id, isValid, errorMessage, onChange, onFinish,
 			label={title}
 			labelAfter
 			required={schema.required}
+			valueIsUpdating={valueIsUpdating}
```
Comment (sgendre) [inline packages/forms/src/UIForm/fields/Select/Select.component.js]: replacing by 'updatingValue' would have the same meaning

Comment (jsomsanith) [reply packages/forms/src/UIForm/fields/Select/Select.component.js]: `would have the same meaning` --> So let's not change it :D

**Pull request merged**